tests/manage/mcg/test_namespace.py

```diff
@@ -27,10 +26,10 @@ class TestNamespace(MCGTest):
 
     MCG_NS_RESULT_DIR = "/result"
     MCG_NS_ORIGINAL_DIR = "/original"
+    # TODO: fix this when https://github.com/red-hat-storage/ocs-ci/issues/3338
+    # is resolved
+    DEFAULT_REGION = "us-east-2"
 
-    # Test is skipped for other platforms due to
-    # https://github.com/red-hat-storage/ocs-ci/issues/2774
-    @aws_platform_required
```

Comment:
I don't see how this will be able to run on Azure, GCP and AWS, as the region names are always bit different. What am I missing?

Reply:
Tests in this file currently use resources from AWS, Azure and RGW. AWS and Azure use same region and RGW is not using a region to authenticate.